examples/gitserver/hooks/detect-language

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# detect-language returns a string indicating the image repository to use as a base
```

Comment:
I think, at some point we should build a small Docker image we will use for code detection based on https://github.com/github/linguist